src/app/file-manager/file-explorer/file-explorer.component.html

```diff
@@ -53,10 +53,6 @@ <h1>Select files to download</h1>
 
 <p-toolbar styleClass="toolbar-accent">
   <div class="ui-toolbar-group-left">
-    <h3 class="subtitle">Selected: {{ fileCount }} file<span *ngIf="fileCount !== 1">s</span>. Total size: {{ totalSize | filesize }} </h3>
```

Comment:
Is this being removed because a more detailed view is being added to replace it?

Reply:
Yes, this will be moved to another place as Nady mention today in the meeting. Also we need this space to put the message 'download in progress..'